deploy.sh

```diff
@@ -1,4 +1,13 @@
 #!/bin/bash -e
 
-source ./ci/git-ssh.sh
-mkdocs gh-deploy --clean
+mkdir _deploy
+cd _deploy
+sd-step exec git 'git clone -b gh-pages git@github.com:screwdriver-cd/guide.git'
+cd guide
+rm -rf *
+cp -R ../../_site/* .
+sd-step exec git 'git add -A'
+sd-step exec git 'git commit -m "Deployed by Screwdriver"'
+sd-step exec git 'git push origin gh-pages'
```

Comment:
Ah, this probably won't work without `source ./ci/git-ssh.sh`.
Inside the git-ssh script, it will use the GIT_KEY to setup ssh for you so that you can push